provider/marathon/marathon_test.go

```diff
@@ -1801,3 +1807,33 @@ func TestParseIndex(t *testing.T) {
 		})
 	}
 }
+
+func TestMarathonGetBasicAuth(t *testing.T) {
+	provider := &Provider{}
+
+	applications := []struct {
```

Comment:
Could you rename `applications` to `cases`?

(I realize you likely just copied this from existing tests; which unfortunately did not pick a better name either.)